src/login-root-saga.js

```diff
@@ -1,7 +1,15 @@
+import { takeLatest } from "redux-saga/effects";
 
+import * as actions from "login/LoginMain/LoginMain_actions";
+import { requestLoginConfig } from "login/LoginMain/LoginMain_sagas";
+
+import * as init_actions from "login/InitResetForm/InitResetForm_actions";
```

Comment:
Maybe we should rethink the naming of the reset password components to be more minute and more descriptive.